qemu/tests/cgroup.py

```diff
@@ -2112,7 +2112,7 @@ def stress_thread():
             time.sleep(10)
             o = process.system_output("top -b -p %s -n 3 | tail -1" % vm.get_pid(),
                                       shell=True).decode()
-            if 49 <= float(o.split()[-4]) <= 51:
+            if 43 <= float(o.split()[-4]) <= 51:
```

Comment:
limited cpu time 50%, so we just need to check cpu utilization less than 51% is enough

Reply:
Updated